Userland/Libraries/LibWeb/Painting/BorderPainting.cpp

```diff
@@ -54,7 +55,35 @@ BorderRadiiData normalized_border_radii_data(Layout::Node const& node, CSSPixelR
     return BorderRadiiData { top_left_radius_px, top_right_radius_px, bottom_right_radius_px, bottom_left_radius_px };
 }
 
-void paint_border(PaintContext& context, BorderEdge edge, DevicePixelRect const& rect, BorderRadiiData const& border_radii_data, BordersData const& borders_data)
+Gfx::Color border_color(BorderEdge edge, BordersData const& borders_data)
+{
+    auto const& border_data = [&] {
+        switch (edge) {
+        case BorderEdge::Top:
+            return borders_data.top;
+        case BorderEdge::Right:
+            return borders_data.right;
+        case BorderEdge::Bottom:
+            return borders_data.bottom;
+        default: // BorderEdge::Left:
+            return borders_data.left;
```

Comment:
nit: General pattern would be to do `case BorderEdge::Left: ` with `default: VERIFY_NOT_REACHED()`.